docs/source/data/errors.mdx

```diff
@@ -332,130 +346,135 @@ You can edit Apollo Server error details before they're passed to a client or re
+- Return `null` to prevent the error from being reported entirely
 
->**For federated graphs**, instead define `rewriteError` in each subgraph's [inline trace plugin](../api/plugin/inline-trace/#rewriteerror). Do not define it in the gateway.
+> **For federated graphs**, instead define `rewriteError` in each subgraph's [inline trace plugin](/apollo-server/api/plugin/inline-trace/#rewriteerror). Do not define it in the gateway.
 
 #### Example: Ignoring common low-severity errors
 
-Let's say our server is `throw`ing an `AuthenticationError` whenever an incorrect password is provided. We can avoid reporting these errors to Apollo Studio by defining `rewriteError`, like so:
+Let's say our server is `throw`ing an `UNAUTHENTICATED` error whenever a user enters an incorrect password. We can avoid reporting these errors to Apollo Studio by defining `rewriteError`, like so:
+
+```ts
+import { ApolloServer } from '@apollo/server';
+import { ApolloServerPluginUsageReporting } from '@apollo/server/plugin/usageReporting';
 
-```js {7-17}
-const { ApolloServer, AuthenticationError } = require("apollo-server");
-const {
-  ApolloServerPluginUsageReporting,
-  ApolloServerPluginLandingPageLocalDefault,
-} = require('apollo-server-core');
 const server = new ApolloServer({
   typeDefs,
   resolvers,
-  csrfPrevention: true,
-  cache: "bounded",
   plugins: [
+    // highlight-start
     ApolloServerPluginUsageReporting({
       rewriteError(err) {
-        // Return `null` to avoid reporting `AuthenticationError`s
-        if (err instanceof AuthenticationError) {
+        // Return `null` to avoid reporting `UNAUTHENTICATED` errors
+        if (err.extensions.code === 'UNAUTHENTICATED') {
           return null;
         }
 
         // All other errors will be reported.
         return err;
-      }
+      },
     }),
-    ApolloServerPluginLandingPageLocalDefault({ embed: true }),
+    // highlight-end
   ],
 });
 ```
 
-This example configuration ensures that any `AuthenticationError` that's thrown within a resolver is only reported to the client, and never sent to Apollo Studio. All other errors are transmitted to Studio normally.
+This example configuration ensures that any `UNAUTHENTICATED` error that's thrown within a resolver is only reported to the client, and never sent to Apollo Studio. All other errors are transmitted to Studio normally.
 
 #### Example: Filtering errors based on other properties
 
-When generating an error (e.g., `new ApolloError("Failure!")`), the error's `message` is the most common property (in this case it's `Failure!`). However, any number of properties can be attached to the error (such as a `code` property).
+When generating an error (e.g., `new GraphQLError("Failure!")`), the error's `message` is the most common property (in this case it's `Failure!`). However, any number of properties can be attached to the error (such as a `code` property).
```

Comment:
probably best to replace "property" with "extensions" in this section, or to say "the message or extensions".  code isn't really a property, it's an extension.

maybe worth noting somewhere that the error as seen by rewriteError is always a GraphQLError and that you might want to use unwrapResolverError on it.

Reply:
Love it, followed both of these suggestions!